Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -969,6 +973,22 @@ def fetch_incidents():
     demisto.incidents(incidents)
 
 
+def test():
+    """Verify that the first_fetch parameter is according to the standards, if exists.
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+
+    if FETCH_TIME:
```

Comment:
Is this the only parameter you can verify?

Reply:
Yes, the other ones are `creds`, `server URL` and `organization name`